esmvalcore/cmor/_fixes/access/_base_fix.py

```diff
@@ -0,0 +1,34 @@
+"""Fix base classes for ACCESS-ESM on-the-fly CMORizer."""
+
+import logging
+
+from iris.cube import CubeList
+
+from esmvalcore.cmor._fixes.native_datasets import NativeDatasetFix
+
+logger = logging.getLogger(__name__)
+
+
+class AccessFix(NativeDatasetFix):
+    """Fixes functions."""
+
+    def fix_coord_system(self, cube):
+        """Delete coord_system to make CubeList able to merge."""
+        for dim in cube.dim_coords:
+            if dim.coord_system is not None:
+                cube.coord(dim.standard_name).coord_system = None
+
+    def fix_height_value(self, cube):
+        """Fix height value to make it comparable to other dataset."""
+        if cube.coord('height').points[0] != 2:
+            cube.coord('height').points = [2]
```

Comment:
That fix, as fas as I know is valid for Tas not for other variables. Please move it to the Tas class.
See @schlunma previous comment regarding uas etc.

Reply:
Just done it